examples/03_validation_set/plot_1_simple_crossval.py

```diff
@@ -0,0 +1,147 @@
+# %%
+"""
+==================================================================
+Using a validation set to monitor the convergence of the fugw loss
+==================================================================
+
+In this example, we use 3 fMRI feature maps for training and 2 independant
+fMRI feature maps for testing to examine the evolutions of a training and
+a validation loss on 2 low-resolution brain volumes.
+"""
+# sphinx_gallery_thumbnail_number = 9
```

Comment:
This determines which image generated in this example gets to be the thumbnail in the example gallery.
I think the second image (showing the 2 losses) is the most interesting one :)
```suggestion
# sphinx_gallery_thumbnail_number = 2
```